CONTRIBUTING.md

```diff
@@ -21,14 +21,23 @@ pypi. Update your installation as follows.
     $ pip uninstall mne  # uninstall pypi mne
     $ pip install -e .  # use the cloned repo for a local install of mne
 
-Then, install the following python packages:
+Then, install the following python packages for testing purposes:
 
     $ pip install flake8 pytest pytest-cov
 
+Afterwards, also install all optional dependencies of `mne-bids` and their
+dependencies:
+
+    $ pip install scikit-learn nibabel nilean pybv
+
 Finally, it is necessary to install the
 [BIDS validator](https://github.com/bids-standard/bids-validator). The outputs
 of MNE-BIDS are run through the BIDS validator to check if the conversion
 worked.
+You will need the `command line version` of the validator, to be installed via
+[Node.js](https://nodejs.org/en/).
+Please follow the installation instructions as detailed in the README of the
+bids-validator repository.
```

Comment:
I think we need people to sometimes use the development version of the validator. It's better to explicitly link [here](https://github.com/bids-standard/bids-validator/blob/master/CONTRIBUTING.md#using-the-development-version-of-bids-validator) for instructions how to install it.